libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1022,10 +1022,75 @@ const AP_Param::GroupInfo AP_OSD_Screen::var_info2[] = {
     // @Range: 0 15
     AP_SUBGROUPINFO(link_quality, "LINK_Q", 1, AP_OSD_Screen, AP_OSD_Setting),
 
+    // @Param: CRSFPWR_EN
```

Comment:
Doesn't need to be CRSF specific - applies to ELRS too and possibly others

Reply:
ok fixed this also